.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+## Description
+
+Please explain the changes you have made here.
+
+## Jira Ticket
+
+Please add a link to the related Jira ticket (if applicable).
+
+## Checklist
+
+These are items (excluding GitHub Checks) which should be confirmed before a branch is ready to merge.
+
+- [ ] Acceptance Criteria met
+- [ ] Commit messages are meaningful
+- [ ] Manually tested
+- [ ] Self-reviewed your code
+- [ ] Code reviewed from two other developers
```

Comment:
This is handled and enforced by GitHub, so no need to include it in the checklist.